packages/lib/services/spellChecker/SpellCheckerServiceDriverBase.ts

```diff
@@ -4,15 +4,15 @@ export default class SpellCheckerServiceDriverBase {
 		throw new Error('Not implemented');
 	}
 
-	public setLanguage(_v: string) {
+	public setLanguage(_v: any) {
 		throw new Error('Not implemented');
 	}
 
 	public get language(): string {
 		throw new Error('Not implemented');
 	}
 
-	public addWordToSpellCheckerDictionary(_language: string, _word: string) {
+	public addWordToSpellCheckerDictionary(_word: string, _language?: string) {
```

Comment:
Consider documenting the expected behavior if `_language` isn't given (for if a different `SpellCheckerServiceDriver` is added in the future).

Reply:
Don't really now what to document here. The `_language` is optional, if it omitted, it will be set to undefined.

Reply:
Okay! That makes sense.